src/ocamlorg_frontend/pages/outreachy.eml

```diff
@@ -0,0 +1,56 @@
+let render (metadata : Ood.Outreachy.t list)
+= 
+Layout.render
+~title:"OCaml Outreachy Internships"
+~description:"Records Of OCaml Community Past Outreachy Internship Projects"
+~canonical:Url.outreachy @@
+<div class="intro-section-simple">
+    <div class="container-fluid">
+        <div class="text-center prose w-full lg:w-2/3 m-auto">
+            <h1 class="font-bold mb-6">OCaml Outreachy Internships</h1>
+            <p>This is record of all past OCaml Community <a href="https://www.outreachy.org/" target="_blank">Outreachy</a> Internship Projects.</p>
+        </div>
+    </div>
+</div>
+
+<div class="py-12 bg-white">
+  <div class="container relative mx-auto px-4 items-center">
+          <% metadata |> List.iter (fun (rounds : Ood.Outreachy.t) -> %>
+              <div class="block border-b border-[#EF8316] p-6 text-left my-2">
+                  <h2 class="font-bold"><%s rounds.name %></h2>
```

Comment:
```suggestion
              <div class="border-b border-[#EF8316] p-6 text-left my-2">
                  <h2 class="font-bold"><%s rounds.name %></h2>
```

`display:block` is the default style of a `div`, so it's a useless class here.

It also looks to me like the `h2` and the `div` can be collapsed into a single `h2` element.